week2/solutions/RO_flowsheet_with_ERD_build-solution.ipynb

```diff
@@ -0,0 +1,753 @@
+      "    Unit Performance\n",
+      "\n",
+      "    Variables: \n",
+      "\n",
+      "    Key                        : Value   : Units         : Fixed : Bounds\n",
+      "                 Membrane Area :  54.329 :    meter ** 2 : False : (0.1, 100000.0)\n",
+      "    Solvent Mass Recovery Rate : 0.50449 : dimensionless : False : (0.01, 0.999999)\n",
+      "      Volumetric Recovery Rate : 0.50000 : dimensionless :  True : (None, None)\n",
+      "\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                                              Units         Feed Inlet  Feed Outlet  Permeate Outlet\n",
+      "    flow_mass_phase_comp ('Liq', 'H2O')  kilogram / second     0.98796     0.48955        0.49842   \n",
+      "    flow_mass_phase_comp ('Liq', 'TDS')  kilogram / second    0.035000    0.034869     0.00013150   \n",
+      "    temperature                                     kelvin      298.15      298.15         298.15   \n",
+      "    pressure                                        pascal  7.5000e+06  7.4586e+06     1.0132e+05   \n",
+      "====================================================================================\n",
+      "\n",
+      "====================================================================================\n",
+      "Unit : fs.erd                                                              Time: 0.0\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Unit Performance\n",
+      "\n",
+      "    Variables: \n",
+      "\n",
+      "    Key             : Value       : Units         : Fixed : Bounds\n",
+      "         Efficiency :     0.95000 : dimensionless :  True : (None, None)\n",
+      "    Mechanical Work :     -3498.9 :          watt : False : (None, None)\n",
+      "    Pressure Change : -7.3572e+06 :        pascal : False : (None, None)\n",
+      "     Pressure Ratio :    0.013585 : dimensionless : False : (None, None)\n",
+      "\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                                              Units           Inlet     Outlet  \n",
+      "    flow_mass_phase_comp ('Liq', 'H2O')  kilogram / second    0.48955    0.48955\n",
+      "    flow_mass_phase_comp ('Liq', 'TDS')  kilogram / second   0.034869   0.034869\n",
+      "    temperature                                     kelvin     298.15     298.15\n",
+      "    pressure                                        pascal 7.4586e+06 1.0132e+05\n",
+      "====================================================================================\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Report unit-level performance results for (1) the pump, (2) RO and (3) the energy recovery device, using the report() method for each unit on the flowsheet.\n",
```

Comment:
Add a markdown cell before this cell that states: "Step 7: Display the results". Also add this to the base file